beacon_node/beacon_chain/src/data_availability_checker.rs

```diff
@@ -679,8 +679,9 @@ async fn availability_cache_maintenance_service<T: BeaconChainTypes>(
                 let cutoff_epoch = std::cmp::max(
                     finalized_epoch + 1,
                     std::cmp::max(
-                        current_epoch
-                            .saturating_sub(chain.spec.min_epochs_for_blob_sidecars_requests),
+                        current_epoch.saturating_sub(
+                            chain.spec.min_epochs_for_data_column_sidecars_requests,
+                        ),
```

Comment:
`do_maintenance` prunes blocks that are outside the da_window. Now the da_window size is dependant on if holds data columns or blobs. So if:
- block is for epoch < FULU_FORK_EPOCH, `min_epochs_for_blob_sidecars_requests` applies
- block is for epoch >= FULU_FORK_EPOCH, `min_epochs_for_data_column_sidecars_requests` applies

In a way it feels `min_epochs_for_data_column_sidecars_requests` is just `min_epochs_for_blob_sidecars_requests_fulu`.

So in this case the pruning function should conditionally use one value or the other depending on the block epoch